### PR TITLE
Update state proto with additional fields.

### PR DESCRIPTION
**Pull request by michelle192837**

### CODE REVIEW

**Comment by michelle192837** [comment]
Going to clean up the comments for consistency, I'll remove WIP when that's done.

I expect we'll need to rename some fields again but will tackle that separately.

**Comment by michelle192837** [comment]
Still can't get Bazel to find the timestamp proto :/ So leaving it there and reserving the fields for now.

File: pb/state/state.proto [hunk 1]
```diff
@@ -40,21 +136,128 @@ message Row {
...
+  reserved 6;
+
+  // Names of metrics associated with this test case. Stored separate from
+  // metric info (which may be omitted).
+  repeated string metric = 7;
 
   repeated Metric metrics = 8;  // Numerical performance/timing data, etc.
 
-  repeated string icons = 9;  // Put 1-2 chars inside cell (F for fail, etc)
+  repeated string icons = 9;  // 1-5 chars inside cell (F for fail, etc)
```
Comment (fejta) [inline pb/state/state.proto]: woo!

File: pb/state/state.proto [hunk 2]
```diff
@@ -40,21 +136,128 @@ message Row {
...
+
+  // Summaries of dashboard tabs associated with this group.
+  repeated DashboardTabSummary dashboard_tab_summaries = 3;
+
+  // Metrics associated with the update for this group.
+  UpdateInfo update_entry = 4;
+}
+
+// An updater server updates test groups upon request.
+service Updater {
```
Comment (fejta) [inline pb/state/state.proto]: I suspect this is going to import a lot of unnecessary junk. Thoughts on moving update RPCs into their own proto?

Comment (michelle192837) [reply pb/state/state.proto]: That's reasonable yeah, let me go do that.

Comment (michelle192837) [reply pb/state/state.proto]: Done!

**Comment by fejta** [comment]
Maybe create an alpha.2 release if you start pulling this code into somewhere else (k/ti, etc) https://github.com/GoogleCloudPlatform/testgrid/releases

**Comment by michelle192837** [comment]
Noted! Not yet yeah, but probably not far off.

**Pull request merged**